packages/shared/src/components/ScrollToTopButton.tsx

```diff
@@ -36,29 +38,17 @@ export default function ScrollToTopButton(): ReactElement {
   };
 
   return (
-    <>
-      <Button
-        aria-label="scroll to top"
-        {...props}
-        className={classNames(
-          'absolute -top-18 right-4 z-2 laptop:hidden',
-          !show && 'hidden',
-        )}
-        size={ButtonSize.Large}
-        variant={ButtonVariant.Primary}
-        style={style}
-      />
-      <Button
-        aria-label="scroll to top"
-        {...props}
-        className={classNames(
-          'absolute -top-24 right-8 z-2 hidden',
-          show && 'laptop:flex',
-        )}
-        variant={ButtonVariant.Primary}
-        size={ButtonSize.XLarge}
-        style={style}
-      />
-    </>
+    <Button
+      aria-label="scroll to top"
+      {...props}
+      className={classNames(
+        'absolute -top-18 right-4 z-2 laptop:-top-24 laptop:right-8',
+        !show && 'hidden',
+        show && isLaptop && 'laptop:flex',
```

Comment:
Do we need the `isLaptop` check? We want to show the button for all the screen sizes, just different styling right?

Reply:
Is the laptop:flex even doing anything? 🤔

Reply:
🤦‍♀️  no don't think so, nice catch 😅